tests/post_training/conftest.py

```diff
@@ -144,10 +175,14 @@ def accept_pipeline_type(cls, pipeline_type: PipelineType) -> bool:
 
     @classmethod
     @TableColumn.assign_default_value
+    @TableColumn.na_msg
     def get_value(cls, info: Dict[PipelineType, RunInfo], target_pipeline_type: PipelineType) -> str:
-        if info[PipelineType.FP32].FPS > 1e-5:
-            return info[target_pipeline_type].FPS / info[PipelineType.FP32].FPS
-        return "inf"
+        if info[target_pipeline_type].fps is None or info[PipelineType.FP32].fps is None:
+            return None
+        fps = info[target_pipeline_type].fps
+        if fps > 1e-5:
+            return info[target_pipeline_type].fps / info[PipelineType.FP32].fps
```

Comment:
It looks like the FPS difference value which placed in the same table as the common FPS value. It will be confusing.

Reply:
Yes, need check info[PipelineType.FP32].fps > 1e.5.